tools/topology2/include/controls/common.conf

```diff
@@ -0,0 +1,75 @@
+Class.Base."channel" {
+	@args."name" {
+		type "string"
+	}
+
+	DefineAttribute."reg" {}
+
+	DefineAttribute."shift" {}
+
+	attributes {
+		mandatory [
+			"reg"
+			"shift"
+		]
+	}
+
+	reg	1
+	shift	1
+}
+
+Class.Base."ops" {
+	@args."name" {
+		type "string"
+	}
+
+	DefineAttribute."info" {}
+
```

Comment:
Can we define some enums here to help mapping the control to a control type. I think we use int's today but we can use a enum here and the compiler can map it to the int.